charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -57,6 +57,8 @@ defaultDashboards:
     allowCrossNamespaceImport: false
   # -- Create dashboards as ConfigMap despite dependency it requires is not installed
   dashboards:
+    victorialogs:
```

Comment:
Since VM k8s-stack chart doesn't install victorialogs, I don't think we should include victorialogs dashboard here.

Reply:
added vlogs to k8s-stack chart

Reply:
I don't believe we should include victorialogs workload in victoria-metrics-k8s-stack chart:
1. it doesn't integrate with other components, like collecting their logs without other configuration.
2. I think user would prefer to install log and metrics management components separately, since they're not bundled.

I think we could have another new chart for victoriaLogs in the furture, which contains vm-operator, single&cluster version of victoriaLogs, grafana, vmalert&alertmanager and possibly some common optional log collectors.

wdyt @AndrewChubatiuk @zekker6 @f41gh7

Reply:
> I think we could have another new chart for victoriaLogs in the furture, which contains vm-operator, single&cluster version of victoriaLogs, grafana, vmalert&alertmanager and possibly some common optional log collectors.

the only difference between existing k8s-stack chart and suggested one is vmcluster/vmsingle or vlcluster/vlsingle and most of the templates are the same. The only benefit of a separate chart - no need to think about chart size limit, but even this problem could be solved with dashboard subcharts. Since we use a single operator for both VL and VM, many components are expected to be the same for VL chart, I don't see many benefits from having a separate chart. Maybe it makes sense to have one for distributed setup in a future

Reply:
>the only difference between existing k8s-stack chart and suggested one is vlcluster/vlsingle instead of vmcluster/vmsingle and most of the templates are the same. 

No, the collectors(vmagent&log collector) and configurations(vmScrapeConf&logCollectConf) are different, the vmalert configurations(flags&rules) are different. 
If we have the vm-stack and vlogs-stack charts, the only shared resources I can see are grafana and vm-operator.
Except from avoiding copying the shared resources and some templates to the new chart, I see no good thing to have one vm-family chart instead of multiple charts for different stacks. vm-family chart makes the chart values&templates complicated. If users are new to vm or vlogs, or they only want to install one stack, they see a chart with lots of templates and values they don't need and wonder why. Plus we need to test more cases for the vm-family chart, like installing the entire stack and then uninstalling the metrics stack.

Reply:
okay, i can create a separate chart, but first would like to merge https://github.com/VictoriaMetrics/helm-charts/pull/1560 to avoid copying this CRD to a new chart as well

Reply:
@Haleygo 
Updated this PR, moved hack scripts to a repo root hack directory and made dashboards and alerts destination configurable, removed vlogs and added vlogs dashboards and rules to victoria-logs-single chart only